src/DevBetterWeb.Web/Pages/Admin/User.cshtml.cs

```diff
@@ -70,6 +74,21 @@ public async Task<IActionResult> OnGetAsync(string userId)
             RolesNotAssignedToUser = unassignedRoles.Select(x => new SelectListItem(x.Name, x.Id)).ToList();
             Roles = assignedRoles.ToList();
 
+            var subscriptions = await _context.Subscriptions
```

Comment:
this would be something like:
```
var memberSpecification = new SubscriptionsByMemberSpec(userId);
await _repository.List(memberSpecification);
```
Example spec:
https://github.com/DevBetterCom/DevBetterWeb/blob/master/src/DevBetterWeb.Core/Specs/MemberByUserIdSpec.cs

Example repository usage:
https://github.com/DevBetterCom/DevBetterWeb/blob/master/src/DevBetterWeb.Web/Pages/User/MyProfile.cshtml.cs#L62-L63

Reply:
The new spec would go in the Core/Specs folder and would include the Where and OrderBy bits.

Reply:
done